substrate/frame/transaction-payment/src/lib.rs

```diff
@@ -529,7 +529,7 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// Query the detailed fee of a given `call`.
-	pub fn query_fee_details<Extrinsic: sp_runtime::traits::Extrinsic + GetDispatchInfo>(
+	pub fn query_fee_details<Extrinsic: sp_runtime::traits::ExtrinsicLike + GetDispatchInfo>(
```

Comment:
It is not evne clear to me why `Extrinsic: sp_runtime::traits::ExtrinsicLike + GetDispatchInfo`. `Extrinsic: GetDispatchInfo` seem to suffice at first glance.

Reply:
duh, `is_bare()` is used :D